packages/orbit-components/.storybook/README.md

```diff
@@ -0,0 +1,27 @@
+# About
```

Comment:
I created this file to document somewhere information on how to run test-runner and that's important to run this script once the storybook local server is running as well.